src/core_atmosphere/physics/physics_mmm/Makefile

```diff
@@ -7,7 +7,8 @@ dummy:
 
 OBJS = \
 	bl_gwdo.o \
-	bl_ysu.o
+	bl_ysu.o \
```

Comment:
@weiwangncar Since the YSU PBL scheme was merged in PR #1053 and I've rebased this PR on the HEAD of the `develop` branch, I think that line should be present. The important point is that this PR doesn't re-introduce YSU PBL changes.